src/components/Attachments/AttachmentCarousel/attachmentCarouselPropTypes.js

```diff
@@ -20,11 +21,23 @@ const propTypes = {
 
     /** The report currently being looked at */
     report: reportPropTypes.isRequired,
+
+    /** The parent of `report` */
+    parentReport: reportPropTypes,
+
+    /** The specific action that links to the parent report */
```

Comment:
```suggestion
    /** The specific actions that link to the parent report */
```
Unless I'm misunderstanding this. If we're only sending one action, can we send the action itself instead of this container?

Reply:
It's not possible to get the single action via `withOnyx()` so the component must receive all the parent report actions and then lookup the specific report action with `const parentReportAction = parentReportActions[report.parentReportActionID];`

Reply:
I'll update the comment for this prop